rust/main/lander/src/dispatcher.rs

```diff
@@ -11,8 +11,8 @@ pub mod test_utils;
 #[cfg(test)]
 mod tests;
 
+pub use core::*;
```

Comment:
I wonder if we want to expose all the types from every module?

For example, [Sealevel adapter](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/45d0ec5d5f68c98f10d9a48a0f7d5a613c0f1aa7/rust/main/lander/src/adapter/chains/sealevel.rs
) module exposes only two types: `SealevelTxAdapter` and `SealevelTxPrecursor`.